compiler/rustc_expand/src/expand.rs

```diff
@@ -1147,6 +1150,29 @@ impl InvocationCollectorNode for P<ast::Item> {
         collector.cx.current_expansion.module = orig_module;
         res
     }
+    fn declared_names(&self) -> Vec<Ident> {
+        if let ItemKind::Use(ut) = &self.kind {
+            fn collect_use_tree_leaves(ut: &ast::UseTree, idents: &mut Vec<Ident>) {
+                match &ut.kind {
+                    ast::UseTreeKind::Glob => {}
+                    ast::UseTreeKind::Simple(rename) => {
+                        idents.push(rename.to_owned().unwrap_or(ut.ident()))
```

Comment:
```suggestion
                        idents.push(ut.ident())
```
It should already account for renaming.